src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+
+  if (days < 3) {
+    return sum;
+  }
+
+  if (days < 7) {
+    return sum - 20;
+  }
+
+  if (days >= 7) {
+    return sum - 50;
+  }
```

Comment:
```suggestion
  return sum - 50;
```

this `if` is always `true`, otherwise it is not even checked because of previous statement